nodewatcher/modules/analysis/channel_allocation/channel_lookup.py

```diff
@@ -0,0 +1,41 @@
+from nodewatcher.core.monitor import models
+
+
+def get_channel(node_bssid):
+    """
+    Gets the current channel of the BSSID
+
+    :param node_bssid:
+    :return:
+    """
+
+    return models.WifiInterfaceMonitor.objects.filter(bssid=node_bssid)[0].channel
+
+
+def is_2ghz_bssid(node_bssid):
+    """
+    Is this BSSID associated to 2.4ghz or 5ghz?
+
+    :param node_bssid:
+    :return: Boolean whether the BSSID belongs to the 2.4ghz spectrum or not.
+    """
+
+    return models.WifiInterfaceMonitor.objects.filter(bssid=node_bssid)[0].channel <= 11
+
+
+def get_available_channels(node_bssid):
+    """
+    Returns an array of available channels for that node.
+
+    :param node_bssid: BSSID of a node
+    :return: Array of available channels.
+    """
+    highest_2ghz_channel = 11
+    ch_2ghz = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
```

Comment:
Channels for both protocols are already available in the following module, together with some other metadata:
* [nodewatcher.core.generator.cgm.protocols](https://github.com/wlanslovenija/nodewatcher/blob/development/nodewatcher/core/generator/cgm/protocols.py#L221)

If there are some channels missing, add them there and fix this code to use protocol information from there. The same also goes for your `interference` function in `allocation_algorithms`.